src/model/SolvedMaze2d.java

```diff
@@ -0,0 +1,29 @@
+package model;
+
+public class SolvedMaze2d {
```

Comment:
Didn't we have a generic class of 'Solution'?

Reply:
merged to Solution class and deleted this class. Thanks!